### PR TITLE
[STH-2019] add f5 and iZettle as sponsors

### PR DESCRIPTION
**Pull request by krakan**
also EA/Dice were mislabeled as silver sponsors while they want to be community sponsors

### CODE REVIEW

**Comment by phrawzty** [comment]
Hi @krakan: Community sponsorships are generally for small associations, not-for-profits, and other such organisations. EA/Dice is, well, not that. Are you sure about that status change?

**Comment by krakan** [comment]
Hello @phrawzty,
Yes, we'll discuss this in the org team, but for now their status should be "community".

File: data/events/2019-stockholm.yml [hunk 1]
```diff
@@ -108,9 +108,13 @@ sponsors:
   - id: b3init
     level: silver
   - id: ea
-    level: silver
+    level: community
```
Comment (phrawzty) [inline data/events/2019-stockholm.yml]:
```suggestion
    level: silver
```

**Comment by krakan** [comment]
Yes, please do that until we get the potential bronze level sorted out.

**Pull request merged**